shells/electron/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "main": "./index.js",
+  "scripts": {
+    "build": "webpack --config webpack.backend.js && webpack --config webpack.config.js"
+  },
+  "dependencies": {
+    "ws": "^0.7.2"
```

Comment:
`1.1.0` is great (and doesn't require compiling C++ modules)! This PR is rather old, that's why it had `0.7.2`